src/EnergyPlus/DataRuntimeLanguage.hh

```diff
@@ -243,14 +243,14 @@ namespace DataRuntimeLanguage {
         std::string UniqueKeyName; // unique key name associated with output variable
         std::string OutputVarName; // name of output variable
         bool CheckedOkay;          // set to true once checked out okay
-        int Type;                  // type of output var, 1=integer, 2=real, 3=meter
-        int Index;                 // ref index in output processor, points to variable
-        int VariableNum;           // ref to global variable in runtime language
-        int SchedNum;              // ref index ptr to schedule service (filled if Schedule Value)
+        OutputProcessor::VarblType VariableType;
+        int Index;       // ref index in output processor, points to variable
```

Comment:
Either `VarType` or `VariableType` is better than `VarblType` IMO.

Reply:
I used VarType first, but it caused a building error (seems like the compiler was confused with another "VarType" somewhere in the code) so I ended up using VarblType. I'll try VariableType. Thank you for your feedback.

Reply:
Aha!